packaging/suse/rpm/mcp-server-trento.spec

```diff
@@ -59,6 +59,9 @@ install -D -m 0755 %{binaryname} "%{buildroot}%{_bindir}/%{binaryname}"
 # Install the systemd unit
 install -D -m 0644 packaging/suse/rpm/systemd/trento-mcp-server.service %{buildroot}%{_unitdir}/trento-mcp-server.service
 
+# Install example configuration file
+install -D -m 0600 packaging/suse/rpm/systemd/trento-mcp-server.config.yaml %{buildroot}/etc/trento/trento-mcp-server.config.yaml
```

Comment:
I think `%{buildroot}%{_distconfdir}/trento/trento-mcp-server.config.yaml` is more appropriate. I don't think we have any distro where `/etc` is not the config folder, but just in case.

Besides, we need to add the new file in `%files`, otherwise it is not included in the final deliverable.
Something like:
```
%files
.... # your current files
%dir %{_distconfdir}/trento
%{_distconfdir}/trento/trento-mcp-server.config.yaml
```

Reply:
Thanks for the info! Really useful; will perform the changes shortly